ChatQnA/benchmark_results.md

```diff
@@ -0,0 +1,161 @@
+| **Batch Size**                    | 256                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+
+## Benchmark Results
+
+| Users | E2E Latency Avg (ms) | TTFT Avg (ms) | TPOT Avg (ms) |
+| ----- | -------------------- | ------------- | ------------- |
+| 256   | 35,034.7             | 1,042.8       | 33.1          |
+| 128   | 20,996.0             | 529.8         | 19.9          |
+| 64    | 16,602.1             | 404.9         | 15.8          |
+| 32    | 14,646.5             | 260.1         | 14.0          |
+| 16    | 13,669.3             | 193.7         | 13.1          |
+| 8     | 13,275.2             | 157.3         | 12.8          |
+| 4     | 13,038.8             | 127.7         | 12.5          |
+| 2     | 13,059.0             | 129.4         | 12.6          |
+| 1     | 12,906.5             | 126.8         | 12.5          |
+
+## Benchmark Config Yaml
+
+<details>
+<summary>Click to Check Benchmark Config Yaml</summary>
+
+```yaml
+deploy:
+  device: gaudi
+  version: 1.3.0
+  modelUseHostPath: /home/sdp/opea_benchmark/model
+  HUGGINGFACEHUB_API_TOKEN: xxx
+  node: [1]
+  namespace: default
+  timeout: 1000 # timeout in seconds for services to be ready, default 30 minutes
+  interval: 5 # interval in seconds between service ready checks, default 5 seconds
+
+  services:
+    backend:
+      resources:
+        enabled: False
+        cores_per_instance: "16"
+        memory_capacity: "8000Mi"
+      replicaCount: [1, 2, 4, 8]
+
+    teirerank:
+      enabled: False
+      model_id: ""
+      resources:
+        enabled: False
+        cards_per_instance: 1
+      replicaCount: [1, 1, 1, 1]
+
+    tei:
+      model_id: ""
+      resources:
+        enabled: False
+        cores_per_instance: "80"
+        memory_capacity: "20000Mi"
+      replicaCount: [1, 2, 4, 8]
+
+    llm:
+      engine: vllm
+      model_id: "meta-llama/Llama-3.1-8B-Instruct" # mandatory
+      replicaCount:
+        with_teirerank: [7, 15, 31, 63] # When teirerank.enabled is True
+        without_teirerank: [8, 16, 32, 64] # When teirerank.enabled is False
```

Comment:
Document states that 8x Gaudi node is used for testing, which can imply to reader that all 8x Gaudi devices were used.   Benchmark config also lists many different replica counts, but results do not state how many replicas are deployed??